omnibus/resources/agent/msi/cal/usercreate.cpp

```diff
@@ -301,11 +301,14 @@ int doesUserExist(MSIHANDLE hInstall, const CustomActionData& data)
     SID_NAME_USE use;
     std::string narrowdomain;
     DWORD err = 0;
-    BOOL bRet = LookupAccountName(NULL, data.getQualifiedUsername().c_str(), newsid, &cbSid, refDomain, &cchRefDomain, &use);
+    const wchar_t * userToTry = data.getQualifiedUsername().c_str();
+
+    BOOL bRet = LookupAccountName(NULL, userToTry, newsid, &cbSid, refDomain, &cchRefDomain, &use);
     if (bRet) {
         err = GetLastError();
```

Comment:
From `LookupAccountName` docs: `If the function succeeds, the function returns nonzero.` I guess this means that `GetError` will never return an error here.

Reply:
yes, that's true.... but given that that code path should never occur (since I'm passing in parameters that in the "success" case should return an error, I thought I'd pull whatever info I can find.